src/NITTA/Intermediate/Functions.hs

```diff
@@ -55,6 +57,17 @@ module NITTA.Intermediate.Functions (
     -- * Internal
     BrokenBuffer (..),
     brokenBuffer,
+    LogicFunction (..),
+    LUT(..),
+    andLut, 
+    orLut, 
+    notLut,
```

Comment:
Do we actually need to export andLut, orLut, notLut? Maybe we can use them in simulation?

Reply:
In Lua transpiler -- use logical and, or, not functions. Do we acrually need andLut, orLut and notLut -- open question. Let see. If not -- just remove them.